pkg/lib/userexport/model.go

```diff
@@ -24,14 +24,89 @@ var RequestSchema = validation.NewSimpleSchema(`
 		"format": {
 			"type": "string",
 			"enum": ["ndjson", "csv"]
+		},
+		"csv": {
+			"type": "object",
+			"properties": {
+				"fields": {
+					"type": "array",
+					"items": {
+						"type": "object",
+						"properties": {
+							"pointer": {
+								"type": "string"
+							},
+							"field_name": {
+								"type": "string"
+							}
+						},
+						"required": [
+							"pointer"
+						]
+					}
+				}
+			}
 		}
 	},
 	"required": ["format"]
 }
 `)
 
+const DefaultCSVExportField = `
+{
+  "fields": [
+		{"pointer": "/sub"},
+		{"pointer": "/preferred_username"},
+		{"pointer": "/email"},
+		{"pointer": "/phone_number"},
+		{"pointer": "/email_verified"},
+		{"pointer": "/phone_number_verified"},
+		{"pointer": "/name"},
+		{"pointer": "/given_name"},
+		{"pointer": "/middle_name"},
+		{"pointer": "/nickname"},
+		{"pointer": "/profile"},
+		{"pointer": "/picture"},
+		{"pointer": "/website"},
+		{"pointer": "/gender"},
+		{"pointer": "/birthdate"},
+		{"pointer": "/zoneinfo"},
+		{"pointer": "/locale"},
+		{"pointer": "/address/formatted"},
+		{"pointer": "/address/street_address"},
+		{"pointer": "/address/locality"},
+		{"pointer": "/address/region"},
+		{"pointer": "/address/postal_code"},
+		{"pointer": "/address/country"},
+		{"pointer": "/roles"},
+		{"pointer": "/groups"},
+		{"pointer": "/disabled"},
+		{"pointer": "/identities"},
+		{"pointer": "/mfa/emails"},
+		{"pointer": "/mfa/phone_numbers"},
+		{"pointer": "/mfa/totps"},
+		{"pointer": "/biometric_count"},
+		{"pointer": "/passkey_count"}
+	]
+}
+`
```

Comment:
I think you need to refactor this code a bit.

Here are what we want to do

1. We want the schema to include all possible pointers as an enum.
2. We want to have a list of default fields when `fields` is unspecified.
3. And we want to use a SINGLE list, to avoid out-of-sync issue.

One way you can do that is to build `RequestSchema` with `SchemaBuilder` in `func init()`

Reply:
as discussed, some traverse pointer like "/identities/0/claims/email" cannot pass enum validation
so skipped this implementation